src/mtx/Command.ts

```diff
@@ -16,9 +16,6 @@ export interface ICommandExample {
 export type TCommandExampleList = Array<ICommandExample>;
 
 
-export const COMMAND_HEADER_SIZE = 2;
```

Comment:
do we need to remove it? it seems it's on it's place

Reply:
fixed